test/pls.jl

```diff
@@ -348,22 +355,41 @@ end
     @test length(σρ.subj.ρ) == 55
     @test iszero(σρ.subj.ρ[46])
     @test σρ.subj.ρ[46] === -0.0
+
+    show(io, BlockDescription(first(models(:sleepstudy))))
+    @test countlines(seekstart(io)) == 3
+    @test "Diagonal" in Set(split(String(take!(io)), r"\s+"))
+
+    show(io, BlockDescription(last(models(:sleepstudy))))
+    @test countlines(seekstart(io)) == 3
+    @test "BlkDiag" in Set(split(String(take!(io)), r"\s+"))
+
 end
 
 @testset "d3" begin
-    fm = updateL!(LMM(@formula(y ~ 1 + u + (1+u|g) + (1+u|h) + (1+u|i)), MixedModels.dataset(:d3)));
-    @test pwrss(fm) ≈ 5.1261847180180885e6 rtol = 1e-6
-    @test objective(fm) ≈ 901641.2930413672 rtol = 1e-6
-    fit!(fm)
-    @test objective(fm) ≈ 884957.5540213 rtol = 1e-6
+    fm = only(models(:d3))
+    @test pwrss(fm) ≈ 5.30480294295329e6 rtol=1.e-4
+    @test objective(fm) ≈ 884957.5540213 rtol = 1e-4
```

Comment:
this is a big change in the objective -- what happened?

Reply:
I'm not sure I understand the comment.  I don't see a difference in the objective.  I'm not sure why I dropped the tolerance.  Right now if I run it on the local machine under julia-1.4.0 I get an objective of 884957.5539700153.  Perhaps I was confusing `rtol` and `atol`.

Reply:
Ah, this was a mistake on my part reading the diff:

```diff
-    fm = updateL!(LMM(@formula(y ~ 1 + u + (1+u|g) + (1+u|h) + (1+u|i)), MixedModels.dataset(:d3)));
-    @test pwrss(fm) ≈ 5.1261847180180885e6 rtol = 1e-6
-    @test objective(fm) ≈ 901641.2930413672 rtol = 1e-6 # <= READ THIS LINE
-    fit!(fm)
-    @test objective(fm) ≈ 884957.5540213 rtol = 1e-6  # <= INSTEAD OF THIS LINE
+    fm = only(models(:d3))
+    @test pwrss(fm) ≈ 5.30480294295329e6 rtol=1.e-4
+    @test objective(fm) ≈ 884957.5540213 rtol = 1e-4
```